### ISSUE TITLE
Adding custom plugin in @nuxt/content-theme-docs

### ISSUE THREAD

**Issue by sa8ab** [nuxt/content#1003]
### Version
@nuxt/content: 0.11.0
nuxt: 2.15.8

### Steps to reproduce
Adding a plugins folder to the project, then adding the plugin file containing the library registration for example:
```
import Vue from 'vue'
import Something from 'something'
import 'something/dist/something.css'
Vue.use(Something)
```
then adding the plugin to nuxt.config
```
export default  theme({
   ...
    plugins: ['./plugins/pluginName.js']
   ...
  }),
```
### What is Expected?
The library to be installed

### What is actually happening?
I get the error of
Plugin not found: (the local path)\node_modules\@nuxt\content-theme-docs\src\plugins\pluginName.js

**Comment by NozomuIkuta** [comment]
@sa8ab 

**Tl;dr**

As shown in the error message, path to your plugin is calculated as a relative path from `\node_modules@nuxt\content-theme-docs\src\` (i.e. `srcDir`), where Theme Doc's `nuxt.config.js` exists.

You have to pass relative path from `rootDir` (i.e. project root) instead, for example:

```typescript
export default {
  plugins: [
    '~~/plugins/pluginName.js'
    // or
    '@@/plugins/pluginName.js' 
  ]
}
```

Note that, if you have changed `rootDir`, the aliases `~~` and `@@` will refer to another place. 

**Details**

Nuxt has 2 options that determines how Nuxt calculate paths: `rootDir` and `srcDir`. 

`rootDir` is `process.cwd()` and `srcDir` is set same as `rootDir` by default, so you might never have set these options in normal Nuxt project (e.g., if you create it with `create-nuxt-app`).

When it comes to Theme Doc, or generally themed Nuxt applications that put [its core files](https://nuxtjs.org/docs/get-started/directory-structure/) in different place than your project root, you need to recognize these 2 options.

Paths to plugins (and other assets such as modules) are calculated/resolved relatively to `srcDir`.
This is why you can't use `./` relative path in this time.

FYR, here is Theme Doc's configuration of `srcDir`:

https://github.com/nuxt/content/blob/b97cf549577cb8ded2ecb3400aaa8e0fb9845f89/packages/theme-docs/src/index.js#L78

**Further readings**

- https://nuxtjs.org/docs/configuration-glossary/configuration-rootdir
- https://nuxtjs.org/docs/configuration-glossary/configuration-srcdir
- https://nuxtjs.org/docs/configuration-glossary/configuration-alias

**Comment by sa8ab** [comment]
Thanks alot @NozomuIkuta for your detailed explanation.
I fixed it by using require.resolve('./pathToFile').
